client_app/src/components/shelter/ShiftDetailsTable.js

```diff
@@ -7,20 +7,33 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faEnvelope, faCheck, faX, faPenToSquare, faTriangleExclamation} from '@fortawesome/free-solid-svg-icons';
 import { useNavigate } from "react-router-dom";
 import { format } from "date-fns";
+import { shiftDetailsAlwaysShowCols,  shiftDetailsColumnsToRender} from "./ShelterConstants.tsx";
 
 export const ShiftDetailsTable = props => {
   const navigate = useNavigate();
   const { 
     onSignUpVolunteersClick , 
-    onSendEmergencyAlertClick, 
+    onSendVolunteerMessageClick, 
     onStatusModalClick,
     stickyHeader,
     headerZIndex,
     } = props;
+  const [isMobileView, setIsMobileView] = useState(window.innerWidth < 430);
+  const colsToRender = isMobileView ? shiftDetailsAlwaysShowCols : shiftDetailsColumnsToRender;
+  const [areColsExpanded, setAreColsExpanded] = useState(false);
+  let prevStartDate = "0/0/0000";
+
+  useEffect(() => {
+    const handleResize = () => setIsMobileView(window.innerWidth < 430);
+    window.addEventListener("resize", handleResize);
+
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
 
```

Comment:
I think the mobile view could be more compressed if you removed more of the empty space with the titles and dates; shifts

Reply:
Spoke with Amal she said it's okay then -- I asked if maybe we should not render name but she said no